Cython/Compiler/ExprNodes.py

```diff
@@ -1791,25 +1768,21 @@ def compile_time_value(self, env):
 
 
 class StringNode(PyConstNode):
-    # A Python str object, i.e. a byte string in Python 2.x and a
-    # unicode string in Python 3.x
+    # In Python 3.x, all strings are Unicode.
```

Comment:
I can't really say right now if there is still a use case for this class, but it feels like it still makes a difference with `language_level=2`. Need to look up the string semantics there. I'd rather keep the class as it is right now.
```suggestion
    # An unprefixed string literal.
```